core/domain/subscription_services_test.py

```diff
@@ -146,10 +147,11 @@ def test_posting_to_feedback_thread_results_in_subscription(self):
             self.viewer_id)
         self.assertEqual(len(thread_ids_subscribed_to), 1)
         full_thread_id = thread_ids_subscribed_to[0]
-        thread_id = feedback_services.get_thread_id_from_full_thread_id(
+        feedback_thread_class = feedback_domain.FeedbackThread
```

Comment:
See above -- I think it's a bit nicer to avoid just aliasing a class, if possible. One less name to keep in context.